doc/writing.md

```diff
@@ -150,14 +148,14 @@ Hello. ++I have added this part++
 
 ### Line break
 
-You can force line breaks in text (carriage return) using `<br>`, which is the
+You can force line breaks in text (carriage return) using `<br>`, similar to
```

Comment:
I prefer the original here. `<br>` isn't *similar* to the HTML line break, it *is* the HTML line break. What do you think?